2025/build-tooling/demo/final/src/App.tsx

```diff
@@ -1,12 +1,194 @@
+import { useCallback, useState, useMemo, useRef } from "react";
+import "@esri/calcite-components/components/calcite-alert";
 import "@esri/calcite-components/components/calcite-shell";
-import "@esri/calcite-components/components/calcite-panel";
+import "@esri/calcite-components/components/calcite-shell-panel";
+import "@esri/calcite-components/components/calcite-list";
+import "@esri/calcite-components/components/calcite-list-item";
+import "@esri/calcite-components/components/calcite-list-item-group";
+import "@esri/calcite-components/components/calcite-navigation";
+import "@esri/calcite-components/components/calcite-navigation-logo";
+import "@esri/calcite-components/components/calcite-progress";
+import "@arcgis/map-components/components/arcgis-features";
+import "@arcgis/map-components/components/arcgis-legend";
 import "@arcgis/map-components/components/arcgis-map";
 
-export function App() {
+import type Polygon from "@arcgis/core/geometry/Polygon";
+import Graphic from "@arcgis/core/Graphic";
+import WebStyleSymbol from "@arcgis/core/symbols/WebStyleSymbol";
+import Point from "@arcgis/core/geometry/Point";
+import Collection from "@arcgis/core/core/Collection";
+import { webMercatorToGeographic } from "@arcgis/core/geometry/support/webMercatorUtils";
+
+import type { ApiKeyManager } from "@esri/arcgis-rest-request";
+import { findPlacesNearPoint } from "@esri/arcgis-rest-places";
+
+type Nil<T> = T | null | undefined;
+
+interface Result<T> {
+  result?: T;
+  error?: Error;
+  loading?: true;
+}
+
+interface ServiceInfo {
+  authentication: ApiKeyManager;
+  endpoint?: string;
+}
```

Comment:
Move to separate interfaces file?

Reply:
If that would make your demo simpler then yes